otx/algorithms/segmentation/adapters/mmseg/tasks/stage.py

```diff
@@ -136,6 +139,19 @@ def configure_classes(self, cfg, task_adapt_op):
         self.org_model_classes = org_model_classes
         self.model_classes = model_classes
 
+    def configure_head(self, cfg):
+        """Change head to custom head to align with otx inteface for class incremental learning."""
+
+        decode_head = cfg.model.get("decode_head", None)
+        auxiliary_head = cfg.model.get("auxiliary_head", None)
+
+        for head in (decode_head, auxiliary_head):
+            if head is not None:
+                # substitute head.type with OTX Head factory function
+                head_type = head.get("type", None)
+                if head_type:
+                    head_type = otx_head_factory
```

Comment:
We need to subtitute head.type as otx_head_factory also if it's none

Reply:
`            if head is not None:
                # substitute head.type with OTX Head factory function
                head_type = head.get("type", None)
                if head_type != "otx_head_factory":
                    head.base_type = head_type
                    head_type = otx_head_factory
                elif head_type is None:
                    head_type = otx_head_factory
`